src/framework/components/collision/system.js

```diff
@@ -175,6 +176,7 @@ Object.assign(pc, function () {
             var data = {
                 enabled: src.data.enabled,
                 type: src.data.type,
+                offset: src.data.offset,
```

Comment:
I think you have to set `offset` here like `halfExtents` in the following line (since they're both `pc.Vec3` types).